app/matplotlib.ipynb

```diff
@@ -339,16 +473,32 @@
/9Lj6+EwHaowOAMFo9XetyI9aMBaKoka2t8aEo41mAMFYKV9dloekMQQydXoDePNoC5/e0Or205zbcWzzkRYAwPULsuNm+wNFRdLNi7Oxv64bm4+04Ntri6c8yGZcDHs+88wzuPbaa9Hd3Q2/34/09AuNQH5+PpqamoZ83pNPPgmtVsv/l5OTM1VFpqiw6bB5+H/b3H44fWNPn9Np8/CdKSAY2ZeiJmvgcdVh9Qx6TP/73D4WNrd/SspGUdT4dfY7XwMcgcnhHeHRF/Q4fXwnHQA6aBsTtzptHuzsDSLXt+SXoqa7DbMzoJaK0NTjwoEoBHiO+Y76r371K9TU1ODJJ58c93MfffRRWK1W/r/m5uYIlJCiIisv+cJeQYNaCpVk7AthspMUEAsvjP7lJ9PAMNTkZSfJQ46r/sfoUPdp5OLe9H0URcWi/uerVCxAmlY2puelqKVQ9UvflUeDj8Wtt4+2giPAorwkFKaool0ciooJcokQV/UGktt8ZOr7kTG59L3P7373O7z99tv49NNPoVAooFAoIBKJ0NHRwc+qNzQ0IDc3d8jnS6VSSKV0XyQV364sz8CpVitYjmBOlnZcy270SgluXpyDui4nDCoJilPVESzp2PgCHLwBFiqpKOpLuGORx8+CIwSKcQzITLVklRQ3L8pBnWn44+rSmWlI18rg8XOYlamBRBTz48IUNW0tLUyGViGG1eXHjDQ1NLKxxUKRiYW4eXEOzrXboJSKMDtTE+GSUgAQYDm4/CzUYWpHCSF8J+SmRXQ2naL6u3FhNl492ISPT3fAvYmFXDJ12/hi9krwD3/4A1599VV8+umn0Ol0/P033XQT/v73v+NnP/sZDh06hNbWVqxevTp6BaWoCBMJBZifmzTh56eqZUhVj212JNKae1zYcqINvgCHolQVrirPmPL9PrHsZIsFn5/rAkcIlhUmY3lRcrSLNKxUjQypmuGPK4GAoWmTKCqOlKVPrJOtlYuxtDB266pEY3J48fbRFji9LDJ1Mly/IHvSAfyONllQ1+WEXCzElRXRSUNFUbFqQa4OuXoFmnpc2F7ZiWumMFVbTE5xtLS04KGHHoLFYsHatWsxb948LF26FADwm9/8Bvv27cOMGTNw11134ZVXXqER3ykqTuyv7YYvENzPWGt0oNnsinKJYgchBLvPd/F7v7+s64bbx0a5VBRFUVQsOVTfA6c32Da0WTyo6rBP+jXfPhoMIrdxTnrIVgaKooIBfK+dF+ycv3esdUrfOybPxuzsbJB+gYr6S0tLwyeffDLFJYqe5h4XH9xjTWkqcmhu02njbJsNB+u7IRMLcdmstJhNb9Vl9+LTyk54/SyWFxlQmj788vqBUWRpVNkLGIaBUCCAnw1egAkYBoIYGUr1+Fl8crYTJrsXM9JUWDUjBXurTTjfaUeySoL1s9PR3OPCFzUmSERCXDozdcTZdooKN1+Aw/azneiweVCYosSakhS6tWYcHN4APj7dAavbj/Js7aAsDrFqX40J5zqC9dDls9KndElqtAxsN0XCyR3nfpbDR6c7AACb5mdN6rUoKlFdOy8Lf/6sBrvOd6HH6YN+DFkxwiFGLgOpoXAcwfsn22By+GBy+PD+ybZhBzCoxGLz+LH9bDCfabvVg4/PdEa7SMPadrodHVYPzC4/Pj7TAYd3+Kj0q0tTkKQQQyRgsCAvCdlJdOCpv/Wz0yCXCCERCbBuZiqkoti46Nxf141aowNWtx+HG8zYUdmJQw09sLr9qOty4vOqTnx0ugNmlx+dNg9/0UdRU+VgfQ/Od9phc/txvMmCs+00Het47KwyoqnHBavbj73VJj4laCyr63LgQP2FemhvjSnaRZoSy4uSkaqRQihgUJauRskkY898UWNCj9MHg0qCFTG83Yqioqk4VYU5WRoEOIKtp9qn7H1jckadCmIJCUl74vVzYDky6dFTKvb1BRTr4/IFwHIEhxt6YHH7MStDEzOrK9z+C8uzWY7A62f5pXOdNg9ONFuglIqwOF8Pg0qKu1YWRKuoMa8wRYX7V8detN2BS/AtrtBUazZ38Pjs45rgkv3Gbic2H26BRCTAbUtzYFDRWXlqbPrXQ8DgY5YamdMbQIvZBW+Ag0ElhXscaUCjZeBv7priMp9ps6LF7EaWTo45Wdope1+1TIzbl+aF7fW2nGgDAFxRngHRJPe6U1Qi2zQvC6dbbXjvWCvuWBa+c3Ak9IyMYWKhAPNzdfztBXlJtBKdJlJUUhSmBNPcMAywpECPvTUm7Kvtxtk2G9451oruMea5jbRF+Xr0rTAtTlXxy4Ec3gDePNKCM202HKzvwY7K2F0VQI1sXo6Oj9qepBDjkrJU/neWiARYWRy65WFJwfiDH3p8Afzig7M41NCDL2pMePLDc+EpPDUtzM3WQioOHqMauRhlGTT6+HhwBGi1uNFl96LF7IZ6jFHfo6koRYVkVbAeEgsZLJhE0NXxOt9pxydnOnG2zYbtZztxriM+V3B4/Cw+6V2xN5UBsigqHl09NxMMAxxuNKO5Z2piLNEZ9QlqMQeXiOUnK6GMYOCNNaWpmJWpAQMGKerY3KNMhR/DMLhmbiY6bV7IxALoFBK8cfhC/kaWIzA5fGHft+72sagzOaCWipE7IDd2l92LTpsHWTo5kvrtzVmQm4SCZCW8AQ5pGim/L9Ts9PGB44Dg7DoVnzJ1cty1Ih9Wtx8GlRQSkQC3LsmFyeGFVi6GUipClk6OhXlJEAsFY967Zff40dTjQpJCApYQ2D0XZsTarfF1vPQ4fNhb24VUlQzL6PLREBxHUNPlACHBwbxIxKZI1chw14p8WFx+JKskMbNtJF4IGQbzcnTwsxwUEhGsbj/SJhhnwmj3wGjzIjtJDp1i4vs4LS4fWsxupKqlQ8a8kImFuHVJLrrsXmjk4ikNgjawPeuweiYcNT+aPj9nhMMbQJZOPqUDHRQVj9I0MqwoSsYXNd3YcqIN315bHPH3pB31CTjVYsWnvbODapk
```

Comment:
Use relative path instead of absolute. Because in others computers it will not work